internal/settings.go

```diff
@@ -12,14 +12,16 @@ import (
 	"os"
 	"strconv"
 
+	"cloud.google.com/go/auth"
 	"golang.org/x/oauth2"
 	"golang.org/x/oauth2/google"
 	"google.golang.org/api/internal/impersonate"
 	"google.golang.org/grpc"
 )
 
 const (
-	newAuthLibEnVar = "GOOGLE_API_GO_EXPERIMENTAL_USE_NEW_AUTH_LIB"
+	newAuthLibEnVar         = "GOOGLE_API_GO_EXPERIMENTAL_USE_NEW_AUTH_LIB"
```

Comment:
Should this be "...ENABLE_NEW_AUTH_LIB" for symmetry with "DISABLE" in the other env var?